src/libstd/lib.rs

```diff
@@ -247,6 +247,7 @@
 #![feature(cfg_target_thread_local)]
 #![feature(char_error_internals)]
 #![feature(clamp)]
+#![feature(lerp)]
 #![feature(concat_idents)]
```

Comment:
This needs to be put in sorted order per the NB at the top and bottom of this feature list.
<img alt="abcdefghijklmnopqrstuvwxyz" width="60%" src="https://user-images.githubusercontent.com/1940490/83100408-c295ea80-a064-11ea-8cde-42230086445f.png">